packages/discovery-react-components/src/components/DiscoverySearch/DiscoverySearch.tsx

```diff
@@ -107,12 +109,13 @@ export interface SearchContextIFC {
   componentSettings: DiscoveryV2.ComponentSettingsResponse | null;
   isResultsPaginationComponentHidden: boolean | undefined;
   fieldsStore: FieldsStore;
+  globalAggregationsResponseStore: GlobalAggregationsResponseStore;
 }
 
 export interface SearchApiIFC {
   performSearch: (searchParameters: DiscoveryV2.QueryParams, resetAggregations?: boolean) => void;
   fetchAutocompletions: (nlq: string) => Promise<void>;
-  fetchAggregations: (searchParameters: DiscoveryV2.QueryParams) => Promise<void>;
+  fetchGlobalAggregations: (searchParameters: DiscoveryV2.QueryParams) => Promise<void>;
```

Comment:
im wondering if its worth it to change the name of this api because technically it would be a breaking change to this components repo (i doubt anyone is actually using it). I can be convinced to keep it this way for the sake of clarity, but just think its good to weigh the following:
1. keep this named `fetchAggregations` but have the conflict in meaning within the codebase (really it refers to global aggregations)
2. change it to `fetchGlobalAggregations` and do not do a major version increase (could break a client unexpectedly)
3. change it to `fetchGlobalAggregations` and do a major version increase (requires a migration guide when we publish the major version, capturing this commit as such)
i'm open to any options, want to hear your opinion

Reply:
Good point, thanks. I think in this case that I lean toward 1, as I agree it is probably not worth it for this potentially breaking change and I think the meaning is still clear without this name change. Updated.